webapp/components/Editor/defaultExtensions.spec.js

```diff
@@ -0,0 +1,88 @@
+import defaultExtensions from './defaultExtensions.js'
+import { Editor } from 'tiptap'
+
+let content
+let createEditor
+
+describe('defaultExtensions', () => {
+  describe('editor', () => {
+    createEditor = () => {
+      const componentStub = {
+        placeholder: 'placeholder',
+        $t: jest.fn(),
+        $apollo: {
+          mutate: jest.fn(),
+        },
+      }
+      return new Editor({
+        content,
+        extensions: [...defaultExtensions(componentStub)],
+      })
+    }
+  })
+
+  it('renders', () => {
+    content = ''
+    expect(createEditor().getHTML()).toEqual('<p></p>')
+  })
+
+  describe('`content` contains a mentioning', () => {
+    beforeEach(() => {
+      content =
+        '<p>This is a post content mentioning <a class="mention" href="/profile/f0628376-e692-4167-bdb4-d521de5a014f" target="_blank">@alicia-luettgen</a>.</p>'
```

Comment:
`data-mention-id`?

Reply:
is it added in `toDom`?

Reply:
Since `parseDOM` is returning empty array, `Mention` and `Hashtag` never get parsed from html. It's a one-way, let's say "write-only".

Reply:
By the way: I was worried I could break the hashtag implementation if hashtags are not parsed:
1. Tag a post with 3 hashtags
2. Edit the post without touching the hashtags
3. Are the hashtags gone now? - No, because the backend implementation searches for `a[href]` with a `href` starting with `/search/hashtag/`.

It's sometimes really, really confusing because all the implementations are interconnected. We have to be very careful about refactoring it.